tests/attr/test_layer_conductance.py

```diff
@@ -20,6 +20,15 @@ def test_simple_input_conductance(self):
         inp = torch.tensor([[0.0, 100.0, 0.0]])
         self._conductance_test_assert(net, net.linear0, inp, [[0.0, 390.0, 0.0]])
 
+    def test_simple_input_with_scalar_baseline_conductance(self):
+        net = BasicModel_MultiLayer()
+        inp = torch.tensor([[0.0, 100.0, 0.0]])
+        cond = LayerConductance(net, net.linear0)
+        attributions = cond.attribute(inp, 0.0001, target=0)
+        assertArraysAlmostEqual(
+            attributions.squeeze(0).tolist(), [-0.00039, 390.0, -0.00039]
```

Comment:
nit: Could maybe make the scalar baseline a little larger, since this expected conductance is within the delta of 0.05?